.github/workflows/basic.yml

```diff
@@ -71,5 +71,4 @@ jobs:
         run: pnpm install --frozen-lockfile
 
       - name: Run Tests
-        run: pnpm test
-        working-directory: packages/create-catalyst
+        run: pnpm --filter "./packages/cli" --filter "./packages/create-catalyst" test
```

Comment:
We might want to create a turborepo task instead of using the built-in pnpm filters so we can cache test runs.

Reply:
Done! Got a cache hit locally:
<img width="420" alt="Screenshot 2025-07-08 at 12 11 53 PM" src="https://github.com/user-attachments/assets/35f04866-837a-460d-aff9-9883443a5454" />

Feel free to amend the PR if you want to add outputs, I'm less familiar with that area